### PR TITLE
bpo-45122: Remove PyCode_New from mypyc

### PR DESCRIPTION
**Pull request by corona10**
### Description

Guido plans to remove PyCode_New from CPython CAPI.
See: https://bugs.python.org/issue45122

closes: https://github.com/python/mypy/issues/11066

### CODE REVIEW

File: mypyc/lib-rt/exc_ops.c [hunk 1]
```diff
@@ -189,38 +189,6 @@ void CPy_TypeError(const char *expected, PyObject *value) {
     }
 }
 
-// These functions are basically exactly PyCode_NewEmpty and
-// _PyTraceback_Add which are available in all the versions we support.
```
Comment (JelleZijlstra) [inline mypyc/lib-rt/exc_ops.c:193]: This comment still applies to the next function I believe. Not sure what optimization this is talking about.

**Comment by gvanrossum** [comment]
Note that this fixes mypy issue #11066.

**Review by JelleZijlstra — APPROVED** [review]

**Review by 97littleleaf11 — APPROVED** [review]
LGTM!

**Review by gvanrossum — APPROVED** [review]
Is this waiting for someone or something?

**Comment by gvanrossum** [comment]
Thanks! Now I can sleep at night. :-)

**Comment by JelleZijlstra** [comment]
Always happy to be of service.

**Pull request merged**